### PR TITLE
main Release 0.3.0 

### PR DESCRIPTION
**Pull request by g-krisztian**
Update main branch to Release 0.3.0

### CODE REVIEW

**Review by soulflyer — APPROVED** [review]
Looks good

File: project.clj [hunk 1]
```diff
@@ -15,11 +14,11 @@
                  [duct/server.http.jetty "0.2.1"]
```
Comment (baskeboler) [inline project.clj:14]: are we using the duct module or the regular jetty ring adapter?

Comment (g-krisztian) [reply project.clj:14]: We are using duct, it has the :handler key extension for jetty

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+version: "3.3"
+
+services:
+  postgres:
+    build:
+      context: .
+      dockerfile: postgres.dockerfile
+    # environment:
```
Comment (LeaveNhA) [inline docker/docker-compose.yml:8]: Carious on why is it commented? Will dig into it.

File: docker/sql-scripts/init.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
+--;;
+DROP DATABASE IF EXISTS framework;
```
Comment (LeaveNhA) [inline docker/sql-scripts/init.sql:2]: Isn't this should be `Xiana`?

File: script/auto.sh [hunk 4]
```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/env bash
```
Comment (LeaveNhA) [inline script/auto.sh:1]: I love this file!

**Review by soulflyer — APPROVED** [review]

File: src/xiana/core.clj [hunk 5]
```diff
@@ -3,16 +3,34 @@
     [cats.core :as m]
     [cats.monad.either :as me]))
 
+;; state/context record definition
 (defrecord State
```
Comment (LeaveNhA) [inline src/xiana/core.clj:7]: We need to rename it. Right?

**Review by LeaveNhA — APPROVED** [review]
I approve.

**Pull request merged**